### PR TITLE
Fixes #13951 - Add smart proxy plugin to katello scenario

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by ehelms** [comment]
Does this also  need to be included in the capsule scenario? https://github.com/Katello/katello-installer/blob/master/config/capsule-answers.yaml

**Comment by ares** [comment]
probably yes, added

File: config/capsule-answers.yaml [hunk 1]
```diff
@@ -3,3 +3,4 @@ certs:
   generate: false
   deploy: true
 capsule: true
+"foreman_proxy::plugin::remote_execution::ssh": false
```
Comment (ehelms) [inline config/capsule-answers.yaml]: Sorry to nitpick, just not sure if it makes a difference: here you have it in quotes, while in katello-answers you don't.

**Comment by ares** [comment]
removed quotes :-)

**Comment by ehelms** [comment]
ACK - thanks @ares !

**Pull request merged**